database/src/main/scala/cromwell/database/slick/MetadataComponent.scala

```diff
@@ -82,6 +82,14 @@ trait MetadataComponent {
     } yield m
   )
 
+  val workflowExists = Compiled(
+    (workflowExecutionUuid: Rep[String]) => (for {
+      m <- metadata
+      if m.workflowExecutionUuid === workflowExecutionUuid
+      if m.key === WorkflowMetadataKeys.SubmissionTime
```

Comment:
Oh right.  I didn't originally have the `exists` on this so I was doing a clunky `headOption.isDefined` in the caller, but you're absolutely right this doesn't need to consider the key now.  That might also clean up the `CromwellApiServiceSpec` test a bit.